common/views/components/Tracker/Tracker.js

```diff
@@ -13,6 +13,10 @@ type RelevanceRatingData = {|
   query: string,
   page: number,
   workType: ?(string[]),
+  productionDatesTo: ?string,
+  productionDatesFrom: ?string,
+  sort: ?string,
+  sortOrder: ?string,
```

Comment:
This is all stored in the query part of the tracking, we should only be storing event specific information here.